browserslist

```diff
@@ -0,0 +1,5 @@
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+IE 11
```

Comment:
Hadn't seen this file before; seems like a cool tool. Can you describe what would happen when building, testing and deployment would look like if we didn't have this configuration

Reply:
Sure thing! This just helps @babel/preset-env [know](https://babeljs.io/docs/en/babel-preset-env#browserslist-integration) how to transform code for supporting your desired browsers. In this case we need babel to transpile the web component related dependencies (Pharos, lit) to a format supported by IE, to prevent it from crashing in the browser.

Reply:
@michael-iden `browserlist` is also used as part of the JSTOR assets build process, using the default configuration.